src/frontends/tensorflow/src/graph_iterator_saved_model.hpp

```diff
@@ -85,68 +85,97 @@ class GraphIteratorSavedModel : public GraphIteratorProto {
         bool res = m_saved_model->ParseFromIstream(&sm_stream);
         FRONT_END_GENERAL_CHECK(res && m_saved_model->meta_graphs_size(), "Saved Model cannot be parsed");
 
+        const ::tensorflow::MetaGraphDef* meta_graph_with_no_tags = nullptr;
+
         for (const auto& meta_graph : m_saved_model->meta_graphs()) {
```

Comment:
minor:
please add a comment above `for`:
`SavedModel can contain several MetaGraph with different tags. Look for MetaGraph with the required tag`

Reply:
ok